README.md

```diff
@@ -1,9 +1,12 @@
 # InferSharp
 
-**InferSharp** (also referred to as Infer#) is an interprocedural and scalable static code analyzer for C#. Via the capabilities of Facebook's [Infer](https://fbinfer.com/), this tool detects race conditions, null pointer dereferences and resource leaks. Read more about our approach in the [Wiki page](https://github.com/microsoft/infersharp/wiki/InferSharp:-A-Scalable-Code-Analytics-Tool-for-.NET).
+**InferSharp** (also referred to as Infer#) is an interprocedural and scalable static code analyzer for C# with [VSCode integration](https://marketplace.visualstudio.com/items?itemName=matthew-jin.infersharp-ext). Via the capabilities of Facebook's [Infer](https://fbinfer.com/), this tool detects race conditions, null pointer dereferences and resource leaks. Read more about our approach in the [Wiki page](https://github.com/microsoft/infersharp/wiki/InferSharp:-A-Scalable-Code-Analytics-Tool-for-.NET).
```

Comment:
I think it might be good to move this line in front of the demo gif.

We can highlight it by rephrasing it to something like:

### **New: Now InferSharp comes as a [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=matthew-jin.infersharp-ext). Try it out!**

Reply:
Edited